pages/manage_and_operate/observability/logs_data_platform/upgrade_upgrade_to_opensearch_2/guide.en-gb.md

```diff
@@ -0,0 +1,45 @@
+---
+title: OpenSearch Upgrade to 2.X
+excerpt: Learn about the upgrade of OpenSearch from 1.3 to 2.x and its impact on the Logs Data Platform
+updated: 2024-06-05
+---
+
+## Objective
+
+This guide will help you to understand what the upcoming upgrade of OpenSearch from 1.3 to 2.X means for you and which actions you must take.
+
+## Why do we upgrade ?
+
+OpenSearch is one of the core components of the Logs Data Platform, allowing you to index logs at scale while providing the most complete query system for logs available today. The first release of OpenSearch 2 appeared in 2024. Since then, we have been monitoring the project. Now that the 1.X branch of OpenSearch will not have any new features, we need to upgrade our stack to this version to allow the platform to continue evolving. OpenSearch 2.X will bring performance improvements and new ways to interact with your logs.
+
+## OpenSearch API 2.X Breaking changes
+
+The main change for OpenSearch 2.X in terms of API is the [removal of type parameters](https://opensearch.org/docs/latest/breaking-changes/#200){.external}. This means you cannot use the "\_type" meta field anymore. This field has been fixed to the value "\_doc" and all official OpenSearch libraries have been updated to accommodate this change for some time now. The APIs using "\_doc" in their path have not changed, making the upgrade straightforward.
```

Comment:
```suggestion
The main change for OpenSearch 2.X in terms of API is the [removal of type parameters](https://opensearch.org/docs/latest/breaking-changes/#200){.external}. This means you cannot use the `\_type` meta field anymore. This field has been fixed to the value `\_doc` and all official OpenSearch libraries have been updated to accommodate this change for some time now. The APIs using `\_doc` in their path have not changed, making the upgrade straightforward.
```

Reply:
looks good to me, i think your suggestion is better. Thank you !